src/chrome/content/rules/Wind.it.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Wind.it">
+	<target host="wind.it" />
+	<target host="www.wind.it" />
```

Comment:
Please see also:
```
start.wind.it
telefonino.wind.it
startpre.wind.it
wep.wind.it
windcloud.wind.it
windsmartcontrol.wind.it
web.pos.wind.it
authproxy.wind.it
windecare.wind.it
areaclientiaziende.wind.it:8185
windecare.wind.it
partnerportal.wind.it:4445
weg.wind.it
mdm.wind.it

Reply:
More thorough investigation/listing takes more time than what I have available for this, so marked as partial instead.

Reply:
Since I already searched them for you I would be happy if this work wasn't a waste of time.

Reply:
Added mdm and start; for the others I'm not getting http results that I believe would warrant additions.

Reply:
Thanks